### PR TITLE
feat: move mouse events from github

### PR DESCRIPTION
**Pull request by ElviraMustafina**
- separate page for mouse events
- new gifs
- minor updates to code samples: added titles for windows, aligned some elements, removed unused imports

note: the topic is temporarily placed top-level inside "Compose Multiplatform UI framework", I'll move it under "Desktop-specific components" when other parts are ready

### CODE REVIEW

File: topics/compose/compose-desktop-mouse-events.md [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+@OptIn(ExperimentalFoundationApi::class)
+fun main() = singleWindowApplication (title = "Drag") {
+    val windowInfo = LocalWindowInfo.current
+
+    Column {
+        var topBoxOffset by remember { mutableStateOf(Offset(0f, 0f)) }
+
+        Box(modifier = Modifier.offset {
+            IntOffset(topBoxOffset.x.toInt(), topBoxOffset.y.toInt())
+        }.size(200.dp)
```
Comment (m-sasha) [inline topics/compose/compose-desktop-mouse-events.md:372]: Reformatting:
```
        Box(modifier = Modifier
            .offset {
                IntOffset(topBoxOffset.x.toInt(), topBoxOffset.y.toInt())
            }
            .size(200.dp)
```

File: topics/compose/compose-desktop-mouse-events.md [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+        }
+
+        var bottomBoxOffset by remember { mutableStateOf(Offset(0f, 0f)) }
+
+        Box(modifier = Modifier.offset {
+            IntOffset(bottomBoxOffset.x.toInt(), bottomBoxOffset.y.toInt())
+        }.size(200.dp)
+                .background(Color.LightGray)
+                .onDrag(
+                        enabled = true,
```
Comment (m-sasha) [inline topics/compose/compose-desktop-mouse-events.md:388]: What is the point of specifying it explicitly?

File: topics/compose/compose-desktop-mouse-events.md [hunk 3]
```diff
@@ -0,0 +1,550 @@
...
+                }
+        ) {
+            Text(text = "Drag with LMB", modifier = Modifier.align(Alignment.Center))
+        }
+
+        var bottomBoxOffset by remember { mutableStateOf(Offset(0f, 0f)) }
+
+        Box(modifier = Modifier.offset {
+            IntOffset(bottomBoxOffset.x.toInt(), bottomBoxOffset.y.toInt())
+        }.size(200.dp)
```
Comment (m-sasha) [inline topics/compose/compose-desktop-mouse-events.md:385]: Reformat like above.

File: topics/compose/compose-desktop-mouse-events.md [hunk 4]
```diff
@@ -0,0 +1,550 @@
...
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.singleWindowApplication
+
+@OptIn(ExperimentalFoundationApi::class)
+fun main() = singleWindowApplication (title = "Drag") {
+    var topBoxOffset by remember { mutableStateOf(Offset(0f, 0f)) }
+
+    Box(modifier = Modifier.offset {
+        IntOffset(topBoxOffset.x.toInt(), topBoxOffset.y.toInt())
+    }.size(200.dp)
```
Comment (m-sasha) [inline topics/compose/compose-desktop-mouse-events.md:440]: Also reformat

**Review by zamulla — CHANGES_REQUESTED** [review]
Nice job! I left some suggestions.

File: topics/compose/compose-desktop-mouse-events.md [hunk 5]
```diff
@@ -0,0 +1,550 @@
+[//]: # (title: Mouse event listeners)
+
+In your desktop project, you can listen to various mouse events, like clicking, moving, scrolling, entering, and exiting.
```
[the remark this answers is not in the document]
Comment (ElviraMustafina) [reply topics/compose/compose-desktop-mouse-events.md]: Let's try "input region" instead

**Review by zamulla — APPROVED** [review]
Thanks! Approved, just one more suggestion to take into account.

**Pull request merged**